templates/base.html

```diff
@@ -7,7 +7,9 @@
 <link rel=stylesheet href=/static/css/edit.css>
 <script src=/static/js/utils.js></script>
 
-<h1>Cafebabel</h1>
+<a href={{ url_for('home') }} class=logo>
+  <img src=https://rawgit.com/cafebabel/styleguide/master/img/logo-cafebabel.svg alt="logo cafébabel">
```

Comment:
Eventually _logo Cafébabel_ with a capital C, or not?

Reply:
oui ! mieux